core/sail/elasticsearch-store/src/main/java/org/eclipse/rdf4j/sail/elasticsearchstore/ElasticsearchValueFactory.java

```diff
@@ -33,7 +34,7 @@
 	 *
 	 * @return a singleton instance of SimpleValueFactory.
 	 */
-	static ElasticsearchValueFactory getInstance() {
+	public static SimpleValueFactory getInstance() {
```

Comment:
This looks odd. Shouldn't this just have `ElasticsearchValueFactory` as its return type?

Reply:
It should yes, but since I'm overriding a static method Java won't allow it.